modes/parkour/interface.lua

```diff
@@ -587,12 +588,23 @@ onEvent("ChatCommand", function(player, msg)
 
 		save_update = os.time() + 60000 * 3 -- 3 minutes
 		translatedChatMessage("action_within_minute", player)
+			
+	elseif cmd == "map" then
+		
+		newMap()
```

Comment:
No, no and no. You can't check if cmd == whatever and then load a map without making permission checks.

Reply:
fixed now